tests/test_install_fonts.py

```diff
@@ -0,0 +1,28 @@
+from pytest import fail
+from plothist.scripts import install_latin_modern_fonts
+
+
+def test_install_latin_modern_fonts():
+    """
+    Test install_latin_modern_fonts.
+    """
+
+    install_latin_modern_fonts()
```

Comment:
Could you please try the option `--wait=x` with x=2, 5, 10 to the `wget` command in the installation script to see whether the test becomes more stable in that case in case the url is not immediately reachable?